code/modules/reagents/chemistry/machinery/reagentgrinder.dm

```diff
@@ -285,7 +285,9 @@
 	updateUsrDialog()
 
 /obj/machinery/reagentgrinder/proc/get_allowed_by_obj(obj/item/O)
-	return blend_items[O.type]
+	for (var/i in blend_items)
```

Comment:
you can use `is_type_in_list` instead: https://github.com/tgstation/tgstation/blob/9d7cf2533ee981cc03bd7812073bcdec900c852f/code/__HELPERS/_lists.dm#L55

Reply:
Good point, I was just reverting kevinz's change

Reply:
Actually, no. is_type_in_list returns TRUE/FALSE, but get_allowed_by_obj is supposed to return the list value of the type key.

Reply:
oh wait, you are right, I got confused for a moment, yea this way is right